tests/Handler/ArrayCollectionHandlerTest.php

```diff
@@ -48,7 +49,7 @@ public function testSerializeArraySkipByExclusionStrategy()
         $factoryMock = $this->getMockBuilder(MetadataFactoryInterface::class)->getMock();
         $factoryMock->method('getMetadataForClass')->willReturn(new ClassMetadata(ArrayCollection::class));
 
-        $context->method('getExclusionStrategy')->willReturn(new AlwaysExcludeExclusionStrategy());
+        $context->method('getExclusionStrategy')->willReturn(new DisjunctExclusionStrategy([new AlwaysExcludeExclusionStrategy()]));
```

Comment:
repercussion after changing the return type for [Context::getExclusionStrategy()](https://github.com/schmittjoh/serializer/pull/1257/files#diff-3cce1ea0450771adbc3cae800dc672c1edde9f76fc377da115e1ca05c8da926cR111)